store/piecestore/storage/aliyunfs.go

```diff
@@ -113,7 +113,7 @@ func (sc *SessionCache) newAliyunfsSession(cfg ObjectStorageConfig) (*session.Se
 		Region:           aws.String(key.region),
 		Endpoint:         aws.String(endpoint),
 		DisableSSL:       aws.Bool(!disableSSL),
-		S3ForcePathStyle: aws.Bool(true),
+		S3ForcePathStyle: aws.Bool(false),
```

Comment:
You can use [OSS](https://github.com/bnb-chain/greenfield-storage-provider/blob/master/store/piecestore/storage/oss.go), `aliyunfs` will be deprecated.